README.md

```diff
@@ -5,23 +5,54 @@ This repo provides a data exporter by which Kubernetes operators can leverage th
 ## Installation
 
 We provide a helm chart to help install the exporter. Note that this data exporter ONLY retrieves the carbon intensity data from 
-[WattTime](https://www.watttime.org/). You need to get the **authentication ID/Password** from WattTime organization before using the exporter.
+[WattTime](https://www.watttime.org/) OR [Electricity Maps](https://www.electricitymaps.com/). 
+
+### WattTime
+
+You need to get the **authentication ID/Password** from WattTime organization before using the exporter.
+
+```bash
+export WT_USERNAME=XXXX   # WattTime auth info.
+export WT_PASSWORD=YYYY
```

Comment:
I changed the env var names here for consistency with Electricity Maps which are longer.

Reply:
This makes sense.